styleguide-bash.md

```diff
@@ -99,6 +103,20 @@ good example:
 if [ -n "$1" ] && [ -n "$2" ]; then
 ```
 
+## Brackets Notation
+We want to avoid useless whitespace in general, for example in brackets. here is a bad awk example:
+```bash
+awk '{ print $2 }'
+```
+
+and the correct one:
+```bash
+awk '{print $2}'
+```
+
+## Comments in Files
+There are two kinds of comments, those that contain higher level descriptions should be easily and clearly visible --> Empty comment line before and after. Commented code lines are without any empty comment lines.
```

Comment:
And one completely empty line above. I did some formatting experiments, this seems to be the optimum in regards to instantaneous visibility for the reader.